chromium_src/chrome/browser/importer/importer_list.cc

```diff
@@ -41,8 +42,8 @@ void AddChromeToProfiles(std::vector<importer::SourceProfile>* profiles,
 }
 
 void DetectChromeProfiles(std::vector<importer::SourceProfile>* profiles) {
-  base::AssertBlockingAllowed();
-
+  base::ScopedBlockingCall scoped_blocking_call(
```

Comment:
unrelated to this update, is this running on the wrong thread?

Reply:
I think there's a counter issue to remove it.